src/libsyncengine/jobs/jobmanager.h

```diff
@@ -21,17 +21,34 @@
 
 #include "abstractjob.h"
 #include "jobmanagerdata.h"
+
+#include "jobs/network/abstractnetworkjob.h"
+
+#include "network/API_v2/downloadjob.h"
+#include "network/API_v2/upload/upload_session/driveuploadsession.h"
+#include "performance_watcher/performancewatcher.h"
+#include "requests/parameterscache.h"
+
+#include <log4cplus/loggingmacros.h>
+
+#include "jobs/network/networkjobsparams.h"
+
 #include "libcommon/utility/utility.h"
+#include "libcommon/log/sentry/handler.h"
 
-#include <log4cplus/logger.h>
+#include "libcommonserver/log/log.h"
+#include "libcommonserver/utility/utility.h"
 
+#include <Poco/Exception.h>
 #include <Poco/Thread.h>
 #include <Poco/ThreadPool.h>
 #include <Poco/Net/HTTPSClientSession.h>
-#include <Poco/ThreadPool.h>
+
+#include <algorithm> // std::max
 
 namespace KDC {
 
+template<class Job>
```

Comment:
Not an issue, but `Job` further in code looks like a real type and not the template type. This is a personal opinion, but I prefer to use the more conventional `T` in templates.
```suggestion
template<class T>
```

Reply:
Done with 47a97ae00a6ae6094a584b241624cd76a19a99e0 using `J` instead of `Job`.